<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>phraseGeneration</class>
 <widget class="QWidget" name="phraseGeneration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>991</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>595</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">margin-right: 30px; margin-bottom: 15px;
padding-top: 3px; padding-bottom: 3px;
padding-left: 10px; padding-right: 10px;
background-color: rgb(175, 174, 177);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="titleTreeWidget">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>50</y>
     <width>241</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTreeWidget {
    background-color: #555555;
    color: #333333;
}

QTreeWidget::item {
    background-color: #999999;
    color: #000000;
    border-bottom: 1px solid #d3d3d3;
}

QTreeWidget::item:selected {
    background-color: #0078d7;
    color: #ffffff;
}</string>
   </property>
   <property name="headerHidden">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">2</string>
    </property>
   </column>
  </widget>
  <widget class="QPlainTextEdit" name="templateText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>60</y>
     <width>1011</width>
     <height>571</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit {
    padding: 0px 0px 5px 0px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Enter text...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>0</y>
     <width>110</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="add"/>
   </property>
  </widget>
  <widget class="QPushButton" name="toggleTreeButton">
   <property name="geometry">
    <rect>
     <x>413</x>
     <y>3</y>
     <width>102</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Symbol</family>
     <pointsize>27</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="menu"/>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>69</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyButton">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>10</y>
     <width>62</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="templateTitle">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>5</y>
     <width>121</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Enter title...</string>
   </property>
  </widget>
  <zorder>addButton</zorder>
  <zorder>toggleTreeButton</zorder>
  <zorder>deleteButton</zorder>
  <zorder>copyButton</zorder>
  <zorder>templateTitle</zorder>
  <zorder>templateText</zorder>
  <zorder>line</zorder>
  <zorder>saveButton</zorder>
  <zorder>titleTreeWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
